src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -7,6 +7,9 @@
 use Wallabag\CoreBundle\Entity\Entry;
 use Wallabag\CoreBundle\Tools\Utils;
 use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeExtensionGuesser;
+use Symfony\Component\Validator\Constraints\Language as LanguageConstraint;
```

Comment:
Hum, I'll check that.
We don't use the `og:locale` as a fallback for the language field return by graby.